game/Bullet.js

```diff
@@ -5,8 +5,10 @@ try {
   var Character = require('./../game/Character');
 }
 
+Bullet.speed = 0.3;
```

Comment:
0.3, as well as (10 - 9.7) evaluates to 0.3000000000000007.
Probably not what you want.